conjure-java-core/src/integrationInput/java/test/prefix/com/palantir/product/LargeEnumExample.java

```diff
@@ -0,0 +1,1109 @@
+                return visitor.visitE86();
+            case E87:
+                return visitor.visitE87();
+            case E88:
+                return visitor.visitE88();
+            case E89:
+                return visitor.visitE89();
+            case E90:
+                return visitor.visitE90();
+            case E91:
+                return visitor.visitE91();
+            case E92:
+                return visitor.visitE92();
+            case E93:
+                return visitor.visitE93();
+            case E94:
+                return visitor.visitE94();
+            case E95:
+                return visitor.visitE95();
+            case E96:
+                return visitor.visitE96();
+            case E97:
+                return visitor.visitE97();
+            case E98:
+                return visitor.visitE98();
+            case E99:
+                return visitor.visitE99();
+            case E100:
+                return visitor.visitE100();
+            default:
+                return visitor.visitUnknown(string);
+        }
+    }
+
+    public static List<LargeEnumExample> values() {
+        return values;
+    }
+
+    @Generated("com.palantir.conjure.java.types.EnumGenerator")
+    public enum Value {
+        E0,
+
+        E1,
+
+        E2,
+
+        E3,
+
+        E4,
+
+        E5,
+
+        E6,
+
+        E7,
+
+        E8,
+
+        E9,
+
+        E10,
+
+        E11,
+
+        E12,
+
+        E13,
+
+        E14,
+
+        E15,
+
+        E16,
+
+        E17,
+
+        E18,
+
+        E19,
+
+        E20,
+
+        E21,
+
+        E22,
+
+        E23,
+
+        E24,
+
+        E25,
+
+        E26,
+
+        E27,
+
+        E28,
+
+        E29,
+
+        E30,
+
+        E31,
+
+        E32,
+
+        E33,
+
+        E34,
+
+        E35,
+
+        E36,
+
+        E37,
+
+        E38,
+
+        E39,
+
+        E40,
+
+        E41,
+
+        E42,
+
+        E43,
+
+        E44,
+
+        E45,
+
+        E46,
+
+        E47,
+
+        E48,
+
+        E49,
+
+        E50,
+
+        E51,
+
+        E52,
+
+        E53,
+
+        E54,
+
+        E55,
+
+        E56,
+
+        E57,
+
+        E58,
+
+        E59,
+
+        E60,
+
+        E61,
+
+        E62,
+
+        E63,
+
+        E64,
+
+        E65,
+
+        E66,
+
+        E67,
+
+        E68,
+
+        E69,
+
+        E70,
+
+        E71,
+
+        E72,
+
+        E73,
+
+        E74,
+
+        E75,
+
+        E76,
+
+        E77,
+
+        E78,
+
+        E79,
+
+        E80,
+
+        E81,
+
+        E82,
+
+        E83,
+
+        E84,
+
+        E85,
+
+        E86,
+
+        E87,
+
+        E88,
+
+        E89,
+
+        E90,
+
+        E91,
+
+        E92,
+
+        E93,
+
+        E94,
+
+        E95,
+
+        E96,
+
+        E97,
+
+        E98,
+
+        E99,
+
+        E100,
+
+        UNKNOWN
+    }
+
+    @Generated("com.palantir.conjure.java.types.EnumGenerator")
+    public interface Visitor<T> {
```

Comment:
In our internal example, do you know if the visitor is used? I wonder if we should avoid generating the visitor as well, since it'll also be a bit unwieldy.

Reply:
The visitor is not used. However, there's always a chance that some other usage still could use visitors, so I'm hesitant to introduce a back-compat break that'd remove visitors entirely

Reply:
Happy to start small, we can always chip away more later if that turns out to be helpful.